MonstieWash/Assets/Scripts/UI/ScoreUIManager.cs

```diff
@@ -0,0 +1,137 @@
+using JetBrains.Annotations;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.SocialPlatforms.Impl;
+using UnityEngine.UI;
+
+public struct PHTask
+{
+    public string name;
+    public bool complete;
+    public int score;
+
+    public PHTask(string _name, bool _complete, int _score)
+    {  name = _name; complete = _complete; score = _score; }
+}
+
+public class ScoreUIManager : MonoBehaviour
+{
+    [SerializeField] private Image m_clipboard;
+    [SerializeField] private GameObject m_lineItem;
+    [SerializeField] private TextMeshProUGUI m_totalScore;
+    [SerializeField] private GameObject m_strikeOut;
+
+    //Probably temporary, may change to calc scale with number of tasks later.
+    [SerializeField][Range(0.0f, 5.0f)] private float m_spacing;
+
+    /// <summary>
+    /// Placeholder. Suggest Json to player prefs.
+    /// </summary>
+    [SerializeField] private List<PHTask> m_tasks;
+    [SerializeField] private List<GameObject> m_LIList;
```

Comment:
I don't think you'll necessarily get an error for this in C# but I think it's a good habit to initialize lists like this when they are declared.
```suggestion
    [SerializeField] private List<PHTask> m_tasks = new();
    [SerializeField] private List<GameObject> m_LIList = new();
```